guide/README.md

```diff
@@ -23,5 +23,9 @@ If you don't know JS but would like to learn about it, here are a few links to h
 * [Nodeschool, for both JavaScript and Node.js lessons](https://nodeschool.io/)
 * [MDN's JavaScript guide and full documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript)
 * [Google, your best friend](https://google.com)
+* [Grasshopper, a JavaScript learning app by Google](https://grasshopper.app)
+* [w3schools' free tutorials for both server-side and client-side JavaScript](https://w3schools.com)
+* [Learn-js.org's interactive tutorials](https://learn-js.org)
+* [freeCodeCamp where you can learn programming for free (with no ads!)](https://https://www.freecodecamp.org/)
```

Comment:
Not opposed to this one (though it seems to be aimed at covering standardized topics in preparation for coding interviews and standardized certificates), needs a different description(please omit the "(with no ads)" and find something more speaking than "where you can learn programming for free"